styleguide_for_feats

1sourcecontrol opened the pull request:
#237 
What else do I need to change?

1sourcecontrol committed:
styleguide_2nd_passthrough

1sourcecontrol committed:
addec colons, spellchecked

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -106,7 +106,7 @@ must be equal to or less than your strength.
 
 Empty the Mag: 
 Prerequisites:
```
This being blank is going to parse as a section. Maybe we should add the word 'none' after the colon?

1sourcecontrol replied:
sounds good

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -146,84 +146,84 @@ your weapon damages an area, First Shot Counts does not take effect.
 
 Focus Energy: 
 Prerequisites:
```
This is also going to parse as a section head.

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -13,76 +13,76 @@ Passive Feats
 ==============================
```
Can we add a title? Just calling it "Feats" would be helpful.

bleehu replied:
Can we also have a sub-heading the same type as "Active Feats" and "Stat Feats" that says "Passive Feats"? That will make the table of contents make much more sense.

1sourcecontrol replied:
I'm thoroughly confused:

![image](https://user-images.githubusercontent.com/2192156/33745266-a392eb9a-db6a-11e7-86ba-abc08d9159f9.png)

It sounds like you're asking me to put in "Passive Feats" but that already exists; what do you mean? There currently isn't a "Stat Feats" do you want me to add that in somewhere (I don't know it would make sense)?

bleehu replied:
nope, my bad. Git online hides things after 5 unchanged lines in either direction and I just missed it. No change requested.

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -13,76 +13,76 @@ Passive Feats
 an additional +2 miss chance penalty.
 
-Acquired: Additional Plating
+Acquired, Additional Plating:
```
lol! Thanks for fixing this! It was parsing like a stat list!

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -13,76 +13,76 @@ Passive Feats
+Acquired, Additional Plating:
 Prerequisites: Marksman, Gunslinger
-Enables the use of Medium Armour by the listed classes.
+	Enables the use of Medium Armor by the listed classes.
```
I can't thank you enough for writing these out. That had been bugging me for a while.

1sourcecontrol replied:
word

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -13,76 +13,76 @@ Passive Feats
+Acquired, Metal Jacket:
 Prerequisites: Soldier, Engineer, Adept, Berserker
-Enables the use of Full Armour by the listed classes.
+	Enables the use of Full Armor by the listed classes.
```
Again, thank you so much for adding this. You've uncrossed my eyes just a little.

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -13,76 +13,76 @@ Passive Feats
 their next action.
 
 Combat Expertise [Guns]: 
 Prerequisites:
```
This is going to parse as two section headings in a row. See below.

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -232,60 +232,60 @@ attacks. You also get a +10% on rolls to avoid blacking out.
 Shadow Sense:
 Prerequisites: Level 12 Monk with Shadow Walk or Shadow Strike.
-Your Aura Shield gains a +10 DR armour bonus if you have Shadow Strike or a +20 DR armour
+	Your Aura Shield gains a +10 DR armor bonus if you have Shadow Strike or a +20 DR armor
```
Good catch! My eye would have missed this.

1sourcecontrol replied:
Thanks! I thought we had already caught these, but I guess not! lol

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -232,60 +232,60 @@ attacks. You also get a +10% on rolls to avoid blacking out.
 action. HOWEVER, you must shout your Motto or the Name when you perform this 
 move. And hey, keep this move on the DL unless it is absolutely necessary to 
-be used. You wouldn''t want posers to copy it. (Note: this is an RP'd action 
+be used. You wouldn't want posers to copy it. (Note: this is an RP'd action 
```
Another really good catch!

bleehu reviewed (changes requested):
Some very minor tweaks. Great work! Good eye!

1sourcecontrol committed:
adding 'Prerequisites: None'

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -294,320 +294,321 @@ upgrading it with skill points.
+	You are trained in the art of combative tossing, or some other obscure 
 fighting technique.
 
 Unlocked: Boomstick
```
these 'Unlocked: Blah' feats are not going to display as section headings, similar to the ones that you updated well with the 'Acquired:blah' feats.

bleehu replied:
This is not a blocking issue.

bleehu reviewed (approved):
Great work!

1sourcecontrol committed:
fix feat name issues

1sourcecontrol committed:
armor names

A little scope creep for this PR, the armor names weren't updated to C2.0 standards

1sourcecontrol committed:
Merge branch 'dev' into styleguide_2_AL
